src/redux/actions.js

```diff
@@ -0,0 +1,6 @@
+export const XACTIN_LIST_FETCHED = 'LIST/XACTIN_LIST_FETCHED';
```

Comment:
for redux, I can suggest using the ducks approach 
you can read about it here - https://github.com/erikras/ducks-modular-redux

Reply:
Fixed so far